src/lotus/response.rs

```diff
@@ -0,0 +1,198 @@
+//! This file contains the various response types to be used byt the lotus api.
+
+use cid::Cid;
+use fvm_shared::address::Address;
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::MethodNum;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+use strum::{AsRefStr, Display, EnumString};
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct ExecParams {
+    pub code_cid: Cid,
+    pub constructor_params: Vec<u8>,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct InstallActorParams {
+    pub code: Vec<u8>,
+}
+
+#[derive(Debug, Deserialize, Serialize, PartialEq, Eq, Display, EnumString, AsRefStr)]
+pub enum WalletKeyType {
+    #[strum(serialize = "bls")]
+    BLS,
+    #[strum(serialize = "secp256k1")]
+    Secp256k1,
+    #[strum(serialize = "secp256k1-ledger")]
+    Secp256k1Ledger,
+}
+
+pub type WalletListResponse = Vec<String>;
+
+/// Helper struct to interact with lotus node
+#[derive(Deserialize, Serialize, Debug)]
+pub struct CIDMap {
```

Comment:
As mentioned sync, we should have a look at how forest does this. I think there may already be convenient methods to `serde` `cid` and other IPLD data types: https://github.com/ChainSafe/forest/blob/main/ipld/src/json.rs